rust/pact_matching_ffi/src/models/message.rs

```diff
@@ -109,6 +133,16 @@ ffi_fn! {
 
 ffi_fn! {
     /// Get the contents of a `Message`.
+    ///
+    /// # Safety
+    ///
+    /// The returned string must be deleted with `string_delete`.
+    ///
+    /// The returned string can outlive the message.
+    ///
+    /// # Error Handling
+    ///
+    /// If the message is NULL, returns NULL.
```

Comment:
Should we also note that this function can return NULL if the message's contents are `OptionalBody::Missing`?
In that case, no error message will be set, so a caller might have a hard time distinguishing between errors.

Reply:
Ah, that's a fair point.